static_frame/core/store_zip.py

```diff
@@ -34,39 +79,84 @@ def labels(self, *,
                 # always use default decoder
                 yield config_map.default.label_decode(name)
 
-
-class _StoreZipDelimited(_StoreZip):
-    # store attribute of passed-in container_type to use for construction
-    _EXPORTER: AnyCallable
-    _CONSTRUCTOR_ATTR: str
-
     @store_coherent_non_write
     def read_many(self,
             labels: tp.Iterable[tp.Hashable],
             *,
             config: StoreConfigMapInitializer = None,
             container_type: tp.Type[Frame] = Frame,
-            ) -> tp.Iterator[Frame]:
+        ) -> tp.Iterator[Frame]:
 
         config_map = StoreConfigMap.from_initializer(config)
+        multiprocess: bool = config_map.default.read_max_workers is not None
+
+        def gen() -> tp.Iterable[tp.Union[BuildFrameParams, Frame]]:
+            with zipfile.ZipFile(self._fp) as zf:
+                for label in labels:
+                    c: StoreConfig = config_map[label]
+
+                    label_encoded: str = config_map.default.label_encode(label)
+                    src: bytes = zf.read(label_encoded + self._EXT_CONTAINED)
+
+                    hashable_config = HashableConfigParams( # pylint: disable=no-value-for-parameter
+                            index_depth=c.index_depth,
+                            index_name_depth_level=c.index_name_depth_level,
+                            columns_depth=c.columns_depth,
+                            columns_name_depth_level=c.columns_name_depth_level,
+                            columns_select=c.columns_select,
+                            dtypes=c.dtypes,
+                            consolidate_blocks=c.consolidate_blocks,
+                    )
```

Comment:
Maybe `StoreConfig` can have a method `to_store_config_he` or similar that returns a hashable config; that way we encapsulate that extraction.